keylime/cmd/sign_runtime_policy.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import sys
+from typing import NoReturn
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
+
+from keylime.dsse import dsse, ecdsa, x509
+
+# pylint: disable=pointless-string-statement
+"""
+This script signs Keylime runtime policies using DSSE.
+
+Example usage:
+
+To generate an ECDSA key and sign a runtime policy:
+
+```
+python sign_runtime_policy.py -r <runtime policy path> -o keylime-policy-signed.json
+```
+
+To sign a runtime policy with an inline x509 certificate using a user-provided key:
+
+```
+python sign_runtime_policy.py -r <runtime policy path> -k private.pem -b x509 -o keylime-policy-signed.json
+```
+
+To view help and see all available options:
+
+```
+python sign_runtime_policy.py -h
+```
+"""
+
+
+def main() -> None:
+    parser = ConversionParser()
+    parser.add_argument("-r", "--runtime_policy", help="Runtime policy file location", action="store")
+    parser.add_argument("-k", "--keyfile", help="Private key to sign policy with", action="store")
+    parser.add_argument(
+        "-p",
+        "--keypath",
+        help="Path to write created private key if -k is not specified",
```

Comment:
It's actually the **filename** where the key is supposed to be written into.

Reply:
Thanks!